lib/core-integration/src/Test/Integration/Scenario/API/ByronWallets.hs

```diff
@@ -411,17 +413,26 @@ spec = do
     it "BYRON_RESTORE_10 - Can restore wallet using master root key" $ \ctx -> do
         m <- genMnemonics @12
         let passw = "Secure Passphrase"
-        w <- emptyByronWalletWith ctx "random"
+        w1 <- emptyByronWalletWith ctx "random"
              ("Byron Wallet", m, passw)
-        rd <- request
-              @ApiByronWallet ctx (Link.deleteWallet @'Byron w) Default Empty
+        rd1 <- request
+              @ApiByronWallet ctx (Link.deleteWallet @'Byron w1) Default Empty
+        expectResponseCode @IO HTTP.status204 rd1
 
         let (Right seed) = fromMnemonic @'[12] m
-        let _rootXPrv =
+        let rootXPrv = T.decodeUtf8 $ hex $ getKey $
                 generateKeyFromSeed seed (Passphrase $ BA.convert $ T.encodeUtf8 passw)
-
-        expectResponseCode @IO HTTP.status204 rd
-
+        let passwHash =
+                "31347c387c317c714968506842665966555a336f5156434c384449744b\
+                \677642417a6c584d62314d6d4267695433776a556f3d7c53672b436e30\
+                \4232766b4475682f704265335569694577633364385845756f55737661\
+                \42514e62464443353569474f4135736e453144326743346f47564c472b\
+                \524331385958326c6863552f36687a38432f496172773d3d"
+        w2 <- emptyByronWalletFromXPrvWith ctx "random"
+            ("Byron Wallet", rootXPrv, passwHash)
+        rd2 <- request
+              @ApiByronWallet ctx (Link.deleteWallet @'Byron w2) Default Empty
+        expectResponseCode @IO HTTP.status204 rd2
```

Comment:
What's the difference between this scenario and the one in "BYRON_RESTORE_01 - one recipient". To me, it sounds like the latter makes this one redundant (as it includes it already, but has more steps)

Reply:
yes, BYRON_RESTORE_01 is more powerful and check everything what is here, and even more

Reply:
addressed in https://github.com/input-output-hk/cardano-wallet/pull/1476/commits/c8e89edd33a98ab4ef7152dfe5e744b19180706c